README.md

```diff
@@ -82,32 +82,52 @@ specific packages.
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/barrier.svg)](https://repology.org/project/barrier/versions)
 
-### FAQ
+## FAQ - Frequently Asked Questions
 
-Q: Does drag and drop work on linux?
+**Q: Does drag and drop work on linux?**
 
-A: No
+> A: No *(see [#855](https://github.com/debauchee/barrier/issues/544) if you'd like to change that)*
 
-Q: What OSes are supported?
 
-A:
-  - Windows 7, 8, 8.1, and 10
-  - MacOS/OS X
-  - Linux
-  - FreeBSD
-  - OpenBSD
+**Q: What OSes are supported?**
 
-Q: Are 32-bit versions of Windows supported?
+> A: The most recent release of Barrier is known to work on:
+>  - Windows 7, 8, 8.1, and 10
+>  - macOS *(previously known as OS X or Mac OS X)*  
+>    - _The current GUI does **not** work on OS versions prior to macOS 10.12 Sierra (but see the related answer below)_
+>  - Linux
+>  - FreeBSD
+>  - OpenBSD
```

Comment:
This looks fine, but we also seem to be able to support Windows 11, so I'd be grateful if you could append that.